spec/unit/unparser_spec.rb

```diff
@@ -267,6 +267,16 @@ def foo(bar:, baz: "value")
           assert_source '{ :-@ => 1 }'
         end
       end
+
+      context 'method' do
```

Comment:
I though about naming it 'ast contructed manually', but it seemed misleading too. Naming is hard, feel free to advice :)